spec/implementation.rb

```diff
@@ -16,6 +17,33 @@ def configure
 
   private
 
+  def start_remote_server
+    require 'selenium/server'
+
+    @server ||= Selenium::Server.new(remote_server_jar,
+                                     :port       => Selenium::WebDriver::PortProber.above(4444),
+                                     :log        => !!$DEBUG,
+                                     :background => true,
+                                     :timeout    => 60)
+
+    @server.start
+    at_exit { @server.stop }
+  end
+
+  def remote_server_jar
+    require 'open-uri'
+    file_name = "selenium-server-standalone.jar"
+    return file_name if File.exist? file_name
+
+    open(file_name, 'wb') do |file|
+      file << open('http://goo.gl/qTy1IB').read
```

Comment:
Let's make this an environment variable so that it can be overridden (and more explicit).